frontend/src/components/Psychologist/Appointments/NewAppointment.jsx

```diff
@@ -38,6 +37,12 @@ const NewAppointment = () => {
     });
   };
 
+  const patientsMap = patients.map(patient => (
+    { value: `${patient.id}`, label: `${patient.lastName} ${patient.firstNames}` }
+  ));
+  const defaultString = [{ value: '', label: '--- Selectionner un patient', disabled: true, hidden: true }];
+  const concatOptions = defaultString.concat(patientsMap);
```

Comment:
concatOptions is not a very good name
```suggestion
  const allOptions = defaultString.concat(patientsMap);
```